HELM-NOTES.md

```diff
@@ -0,0 +1,196 @@
+# Deploying Apache Hop with Helm on Kubernetes
+
+This document contains technical documentation on how to deploy Apache Hop using Helm on a Kubernetes cluster.
+
+## What is Helm?
+
+Kubernetes can become very complex to manage using standard configuration files (services, pods, volumes, different releases, ... ) which is solved by the use of Helm. Helm is a package manager which can package everything into a simple configurable application. Helm offers easy installing, updating and removal.
+
+> Helm installs charts into Kubernetes, creating a new release for each installation. And to find new charts, you can search Helm chart repositories.
+
+## Prerequisites
+
+The following prerequisites are needed to get started with Apache Hop and Helm.
+
+- A repository which contains a working Hop Docker image
+- A kubernetes cluster (Amazon EKS, Google GKE, minikube, ... )
+- Helm [installed and configured](https://helm.sh/docs/intro/install/)
+
+## The Hop Helm Chart
+
+The following files form the Hop Helm Chart.
+
+```
+hop/
+  Chart.yaml    # A YAML file containing information about the Hop chart
+  charts/       # A directory containing charts which the Hop chart depends on (a database, a queue, ... ).
+  templates/    # A directory which contains the Hop chart definitions which 
+                # generate Kubernetes manifest files based on values
+  values.yaml   # The default Hop configuration values
+```
+
+### Chart.yaml
+
+```
+apiVersion: v2
+name: hop
+description: A Helm chart for Apache Hop
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the Apache Hop Docker image. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# For simplicity this is set to latest but may be changed to reflect different Hop versions.
+appVersion: latest
+```
+
+## Templates and values
+
+The `deployment.yaml` and `_helpers.tpl` files are located in the `templates/` directory and are used with the `values.yaml` file to generate a valid Kubernetes manifest.
+
+### deployment.yaml
```

Comment:
why do we need the full deployment.yaml and values.yaml as code snippet in HELM-NOTES.md?
can we have a target readme.md at chart level with structure as similar to these charts? https://github.com/helm/charts/tree/master/stable

Reply:
Removed the snippets from HELM-NOTES. Added a README for the chart with a notice that there is no official repository yet.